app/assets/javascripts/git_import.js

```diff
@@ -0,0 +1,48 @@
+/* global miqSparkleOn miqSparkleOff showErrorMessage clearMessages */
+
+var GitImport = {
+  retrieveDatastoreClickHandler: function() {
+    $('.git-retrieve-datastore').click(function(event) {
+      event.preventDefault();
+      miqSparkleOn();
+      clearMessages();
+
+      $.post('retrieve_git_datastore', $('#retrieve-git-datastore-form').serialize(), function(data) {
+        var parsedData = JSON.parse(data);
+        var messages = parsedData.message;
+        if (messages && messages.level === "error") {
```

Comment:
@eclarizio - Code climate is asking for single quotes here - That would be consistent with the rest of the file